cli/asc.js

```diff
@@ -718,11 +821,13 @@ exports.main = function main(argv, options, callback) {
   return callback(null);
 
   function readFileNode(filename, baseDir) {
+    let dir = baseDir || sep;
```

Comment:
This still seems off to me. My initial comment was due to Windows absolute paths starting with `C:\` etc., but it might be that `\` does something relative to the current device (like under `cmd`). In that case, `path.sep` would do the same. Now I wonder why this is necessary in the first place as it wasn't necessary before, especially since not specifying a baseDir and then starting at the root seems incorrect.

Reply:
I remember in the past it wasn't always the case that there was a `baseDir` passed.  I just removed it and alas it isn't needed.  I was imagining a case in the browser where your project would be at the root and in my experiences with the compiler in the browser this idea carried over.